numpy/ma/core.py

```diff
@@ -5898,9 +5905,12 @@ def take(self, indices, axis=None, out=None, mode='raise'):
     repeat = _arraymethod('repeat')
     squeeze = _arraymethod('squeeze')
     swapaxes = _arraymethod('swapaxes')
-    T = property(fget=lambda self: self.transpose())
```

Comment:
I'd probably leave this one, it's consistent with the style of the rest of these members